vortex-expr/src/expr.rs

```diff
@@ -43,9 +40,6 @@ fn unbox_any(any: &dyn Any) -> &dyn Any {
     }
 }
 
-#[derive(Debug, PartialEq, Hash, Clone, Eq)]
-pub struct NoOp;
```

Comment:
There was no need to translate it, we will never see it as a datafusion table provider